template.php

```diff
@@ -275,6 +288,17 @@ function europa_form_element(&$variables) {
 
       $output .= $feedback_message;
   }
+  // @todo This would be better handled in a theme_form_element_label override.
+  if (!empty($anchor)) {
+    // Add the link.
+    $output .= $anchor;
+
+    if (!empty($variables['element']['#required'])) {
+      // Find the form-required span and put it after the link.
+      preg_match('/<span class="form-required\b[^>]*>(.*?)<\/span>/is', $output, $required);
```

Comment:
Here I think you could use the preg_replace to avoid doing the preg_match and str_replace.